### DISCUSSION TITLE
Lesson 6: AttributeError: Contract 'FundMe' object has no attribute

### DISCUSSION THREAD

**Discussion by BrysonHall** [smartcontractkit/full-blockchain-solidity-course-py#1213] [Q&A] [answered]
I was doing the Patrick Collins tutorial [link](https://youtu.be/M576WGiDBdQ?t=17887) and I got this error: "AttributeError: Contract 'FundMe' object has no attribute 'account when running  `brownie run scripts/deploy.py --network rinkeby`

Help is much apricated 🙂

My code:

deploy.py

     from brownie import FundMe
     from scripts.helpful_scripts import get_account

    def deploy_fund_me():
        account = get_account()
        fund_me = FundMe.deploy({"from": account})
        print(f"Contract deployed to {fund_me.account}")

     def main():
         deploy_fund_me()

brownie-config.yaml 

    `dependencies:
       # - <organization/repo>@<version>
       - smartcontractkit/chainlink-brownie-contracts@1.1.1
    compiler:
      solc:
        remappings:
            - '@chainlink=smartcontractkit/chainlink-brownie-contracts@1.1.1'

    dotenv: .env
    wallets:
      from_key: ${PRIVATE_KEY}

helpfull_scripst.py 

    from brownie import network, config, accounts

    def get_account():
        if network.show_active() == "development":
            return accounts[0]
        else:
            return accounts.add(config["wallets"]["from_key"])`

FundMe.sol

    `// SPDX-License-Identifier: MIT

     // Smart contract that lets anyone deposit ETH into the contract
    // Only the owner of the contract can withdraw the ETH
    pragma solidity ^0.6.6;

    // Get the latest ETH/USD price from chainlink price feed
    import "@chainlink/contracts/src/v0.6/interfaces/AggregatorV3Interface.sol";
    import "@chainlink/contracts/src/v0.6/vendor/SafeMathChainlink.sol";

    contract FundMe {
        // safe math library check uint256 for integer overflows
        using SafeMathChainlink for uint256;

    //mapping to store which address depositeded how much ETH
    mapping(address => uint256) public addressToAmountFunded;
    // array of addresses who deposited
    address[] public funders;
    //address of the owner (who deployed the contract)
    address public owner;
    AggregatorV3Interface public priceFeed;

    // the first person to deploy the contract is
    // the owner
    constructor(address _priceFeed) public {
        priceFeed = AggregatorV3Interface(_priceFeed);
        owner = msg.sender;
    }

    function fund() public payable {
        // 18 digit number to be compared with donated amount
        uint256 minimumUSD = 50 * 10**18;
        //is the donated amount less than 50USD?
        require(
            getConversionRate(msg.value) >= minimumUSD,
            "You need to spend more ETH!"
        );
        //if not, add to mapping and funders array
        addressToAmountFunded[msg.sender] += msg.value;
        funders.push(msg.sender);
    }

    //function to get the version of the chainlink pricefeed
    function getVersion() public view returns (uint256) {
        return priceFeed.version();
    }

    function getPrice() public view returns (uint256) {
        (, int256 answer, , , ) = priceFeed.latestRoundData();
        // ETH/USD rate in 18 digit
        return uint256(answer * 10000000000);
    }

    // 1000000000
    function getConversionRate(uint256 ethAmount)
        public
        view
        returns (uint256)
    {
        uint256 ethPrice = getPrice();
        uint256 ethAmountInUsd = (ethPrice * ethAmount) / 1000000000000000000;
        // the actual ETH/USD conversation rate, after adjusting the extra 0s.
        return ethAmountInUsd;
    }

    //modifier: https://medium.com/coinmonks/solidity-tutorial-all-about-modifiers-a86cf81c14cb
    modifier onlyOwner() {
        //is the message sender owner of the contract?
        require(msg.sender == owner);

        _;
    }

    // onlyOwner modifer will first check the condition inside it
    // and
    // if true, withdraw function will be executed
    function withdraw() public payable onlyOwner {
        // If you are using version eight (v0.8) of chainlink aggregator interface,
        // you will need to change the code below to
        // payable(msg.sender).transfer(address(this).balance);
        msg.sender.transfer(address(this).balance);

        //iterate through all the mappings and make them 0
        //since all the deposited amount has been withdrawn
        for (
            uint256 funderIndex = 0;
            funderIndex < funders.length;
            funderIndex++
        ) {
            address funder = funders[funderIndex];
            addressToAmountFunded[funder] = 0;
        }
        //funders array will be initialized to 0
        funders = new address[](0);
      }
    }
`

**Comment by cromewar** [comment, marked as answer]
Hello @MrHeim1 
You have this line:
` print(f"Contract deployed to {fund_me.account}")`

A contract does not have something called account, you may be referring to address?

` print(f"Contract deployed to {fund_me.address}")`

**Reply by BrysonHall** [reply]
This solved my issue. Thanks for quick answer.

**Reply by cromewar** [reply]
I'm glad is solved now
